docs/api-guide.md

```diff
@@ -1197,6 +1197,92 @@ In the sample code, the transaction is retried three times maximum and sleeps fo
 
 <div class="notice--info">{{ notice--info | markdownify }}</div>
 
+### Group commit for the Coordinator table
+
+The Coordinator table that is used for Consensus Commit transactions is a vital data store, and using robust storage for it is recommended. However, utilizing more robust storage options, such as internally leveraging multi-AZ or multi-region replication, may lead to increased latency when writing records to the storage, resulting in poor throughput performance.
+
+ScalarDB provides a group commit feature for the Coordinator table that groups multiple record writes into a single write operation, improving write throughput. In this case, latency may increase or decrease, depending on the underlying database and the workload.
+
+To enable the group commit feature, add the following configuration:
+
+#### Configurations
+
```

Comment:
I don't think we need a heading here since we introduce the code block in the previous sentence (`To enable the group commit feature, add the following configuration:`).

```suggestion

```